python/ray/rllib/agents/a3c/a3c.py

```diff
@@ -48,6 +48,10 @@ def get_policy_class(config):
 def validate_config(config):
     if config["entropy_coeff"] < 0:
         raise DeprecationWarning("entropy_coeff must be >= 0")
+    if config["sample_async"] and config["use_pytorch"]:
+        raise ValueError(
+            "The sample_async option is not supported with use_pytorch: "
+            "Multithreading can be lead to crashes if used with pytorch.")
```

Comment:
Nice. Shall we also turn it off by default?

Reply:
Up to you, is it beneficial if used with TensorFlow?